utils/add_copyright.py

```diff
@@ -23,9 +23,11 @@
 import fileinput
 import fnmatch
 import os
+import re
 import sys
 
-COPYRIGHT = 'Copyright 2015 The Shaderc Authors. All rights reserved.'
+COPYRIGHTRE = re.compile('Copyright \d+ The Shaderc Authors. All rights reserved.')
+COPYRIGHT = 'Copyright 2016 The Shaderc Authors. All rights reserved.'
```

Comment:
So if we modify the same file in several years, we add a new copyright line?
This disallows things like
Copyright 2015-2016 ...